flink-dist/src/main/flink-bin/LICENSE

```diff
@@ -254,20 +285,18 @@ under the Apache License (v 2.0):
 
 The Apache Flink project bundles the following components under the MIT License:
 
- - SLF4J (http://www.slf4j.org) - Copyright (c) 2004-2013 QOS.ch
+ - SLF4J (org.slf4j:slf4j-api:1.7.7 - http://www.slf4j.org) - Copyright (c) 2004-2013 QOS.ch
+ - SLF4J-LOG4J Bridge (org.slf4j:slf4j-log4j12:1.7.7 - http://www.slf4j.org) - Copyright (c) 2004-2013 QOS.ch
  - jQuery v2.1.0. (http://jquery.com) - Copyright 2014 jQuery Foundation and other contributors
- - jQuery resize event v1.1 (http://benalman.com/projects/jquery-resize-plugin/) - Copyright (c) 2010 "Cowboy" Ben Alman
```

Comment:
If this dependency is part of the source distribution (which I couldn't find) it shouldn't be removed.

If this dependency is not part of the source distribution but included in the binary distribution because it is a transitive dependency, then it shouldn't be removed.

In any other case, IMO it's ok to remove it.